languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,17 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rule id="PLU_OU_PLUS" name="plu ou plus" default="temp_off">
+          <pattern>
+              <token postag="V.*" postag_regexp="yes"><exception postag="V avoir.*|V etre.*" postag_regexp="yes"></exception></token>
+              <marker>
+                  <token case_sensitive="yes">plu</token>
+              </marker>
+          </pattern>
+          <message>Une erreur semble être détectée dans cette phrase interrogative. Souhaitez-vous écrire</message>
```

Comment:
The message doesn't make sense :) it's not an interrogative sentence. Also, the message contains an unfinished sentence.
You don't need to include "Souhaitez-vous écrire ...", by the way. The user will see the suggestion anyway.
What you could do is just write "Faute de frappe possible trouvée." And if you like, you could explain. "'Plu' est le participe passer de 'plaire', tandis que 'plus' signifie 'davantage' ou exprime que quelque chose ne se déroule plus." or something like that.